js/ui/lookup.js

```diff
@@ -652,7 +644,10 @@ const Lookup = DropDownList.inherit({
                 _fixWrapperPosition: false,
                 width: this._isInitialOptionValue('dropDownOptions.width')
                     ? (function() { return getOuterWidth(this.$element()); }).bind(this)
-                    : this._popupConfig().width
+                    : this._popupConfig().width,
+                height: this._isInitialOptionValue('dropDownOptions.height')
```

Comment:
why? default height should not be equal to lookup input height

Reply:
fixed